api/api.go

```diff
@@ -1012,6 +1028,20 @@ func (serv *MetadataServer) CreateSourceVariant(ctx context.Context, sourceReque
 	return serv.meta.CreateSourceVariant(ctx, sourceRequest)
 }
 
+func extractSourcesFromSQLTransformation(query string) []*pb.NameVariant {
+	numEscapes := strings.Count(query, "{{")
+	sources := make([]*pb.NameVariant, numEscapes)
+	for i := 0; i < numEscapes; i++ {
+		split := strings.SplitN(query, "{{", 2)
```

Comment:
anything in the reflect lib we can use here? probably not but just checking.

Reply:
i'll get rid of this eventually